src/utils.js

```diff
@@ -453,29 +453,19 @@ exports.insertElement = function(elm, doc, target) {
   } catch (e) {}
 };
 
-exports.insertPixel = function (url) {
+exports.triggerPixel = function (url) {
   const img = new Image();
-  img.id = _getUniqueIdentifierStr();
   img.src = url;
```

Comment:
Not I... but I do agree with whoever said it, as long as you're sure it works cross-browser :).

If you don't into the DOM, it shouldn't cause a re-paint of the page, which can have huge performance impact on pub pages.